test/test_infectionsrtfeedback.py

```diff
@@ -39,32 +40,53 @@ def _infection_w_feedback_alt(
     tuple
     """
 
-    Rt = np.array(Rt)  # coerce from jax to use numpy-like operations
     T = len(Rt)
+    Rt = np.array(Rt).reshape(
+        T, -1
+    )  # coerce from jax to use numpy-like operations
     len_gen = len(gen_int)
-    I_vec = np.concatenate([I0, np.zeros(T)])
-    Rt_adj = np.zeros(T)
-
-    for t in range(T):
-        Rt_adj[t] = Rt[t] * np.exp(
-            inf_feedback_strength[t]
-            * np.dot(I_vec[t : t + len_gen], np.flip(inf_feedback_pmf))
-        )
-
-        I_vec[t + len_gen] = Rt_adj[t] * np.dot(
-            I_vec[t : t + len_gen], np.flip(gen_int)
-        )
-
-    return {"post_initialization_infections": I_vec[I0.size :], "rt": Rt_adj}
-
-
-def test_infectionsrtfeedback():
+    I_vec = np.concatenate([I0.reshape(T, -1), np.zeros(Rt.shape)])
```

Comment:
Probably faster and more readable to use`jnp.pad`.

Reply:
Actually, prefer not to pad `I_vec` at all. See later comments.

Reply:
Also, can we have a better name than `I_vec`?

Reply:
> Probably faster and more readable to use `jnp.pad`.

changed to use `np.pad`

Reply:
> Also, can we have a better name than `I_vec`?

renamed in https://github.com/CDCgov/PyRenew/pull/440/commits/97d19d05ed295cd7547376ff4958ea347c743bba